tools/build_config/Jenkinsfile

```diff
@@ -359,6 +369,42 @@ pipeline {
       }
     }
 
+    stage('Build-Docs') {
+      when {
+        expression {env.build_docs?.toBoolean()}
```

Comment:
Should this be `build_docs || push_docs`? Or can we push the docs without building?

Reply:
I went for the opposite way round, if you push docs without building, it will do nothing. Perhaps that was a bad choice.